packages/siri-sx-generator/index.ts

```diff
@@ -50,12 +50,14 @@ const enrichDisruptionsWithOrgInfo = async (disruptions: Disruption[], orgTableN
         .filter(notEmpty);
 };
 
-const convertJsonToSiri = (
+const convertJsonToSiri = async (
     disruptions: Awaited<ReturnType<typeof enrichDisruptionsWithOrgInfo>>,
     currentTime: string,
     responseMessageIdentifier: string,
 ) => {
-    const ptSituationElements = disruptions.map((disruption) => getPtSituationElementFromSiteDisruption(disruption));
+    const ptSituationElements = await Promise.all(
+        disruptions.map(async (disruption) => await getPtSituationElementFromSiteDisruption(disruption)),
+    );
```

Comment:
See comment below, it would be better to make the API call here so we only make it once instead of an API call per disruption.

```suggestion
    const adminAreas = await fetchAdminAreas();
    const ptSituationElements = disruptions.map((disruption) => getPtSituationElementFromSiteDisruption(disruption, adminAreas));
```

Reply:
done